client/state/reader/posts/test/normalization-rules.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External Dependencies
+ */
+import { forEach, repeat } from 'lodash';
+import { expect } from 'chai';
+
+/**
+ * Internal Dependencies
+ */
+import { classifyPost } from '../normalization-rules';
+import * as DISPLAY_TYPES from '../display-types';
+
+function verifyClassification( post, displayTypes ) {
+	classifyPost( post );
+	forEach( displayTypes, displayType => {
+		expect( post.display_type & displayType ).to.equal( displayType );
+	} );
+}
+
+describe( 'normalization-rules', () => {
+	describe( 'classifyPost', () => {
+		it( 'empty post should get UNCLASSIFIED', () => {
+			verifyClassification( {}, [ DISPLAY_TYPES.UNCLASSIFIED ] );
+		} );
+
+		it( 'PHOTO_ONLY', () => {
+			verifyClassification( {
+				canonical_media: {
+					mediaType: 'image',
+					width: 1000
+				},
+				better_excerpt_no_html: repeat( 'no ', 10 )
+			}, [ DISPLAY_TYPES.PHOTO_ONLY ] );
+		} );
+
+		it( 'no PHOTO_ONLY, content too long', () => {
```

Comment:
I'd reword to 'should not classify a PHOTO_ONLY post if the content is too long'